homeassistant/components/waze_travel_time/sensor.py

```diff
@@ -185,46 +194,88 @@ def _resolve_zone(self, friendly_name):
 
         return friendly_name
 
-    @Throttle(MIN_TIME_BETWEEN_UPDATES)
     def update(self):
         """Fetch new state data for the sensor."""
-        import WazeRouteCalculator
-
+        _LOGGER.debug("Fetching Route for %s", self._name)
+        # Get origin latitude and longitude from entity_id.
         if self._origin_entity_id is not None:
-            self._origin = self._get_location_from_entity(
+            self._waze_data.origin = self._get_location_from_entity(
                 self._origin_entity_id)
 
+        # Get destination latitude and longitude from entity_id.
         if self._destination_entity_id is not None:
-            self._destination = self._get_location_from_entity(
+            self._waze_data.destination = self._get_location_from_entity(
                 self._destination_entity_id)
 
-        self._destination = self._resolve_zone(self._destination)
-        self._origin = self._resolve_zone(self._origin)
+        # Get origin from zone name.
+        self._waze_data.origin = self._resolve_zone(
+            self._waze_data.origin)
+
+        # Get desination from zone name.
```

Comment:
desination -> destination

Reply:
Really?  We're going to hold this PR up for a spelling error in a comment.  The repository is unknown at this point because I deleted the fork a month ago when this whole process started.  I no longer have access rights to edit the fork.  Can we please just push this through.

Reply:
I can't do anything (nor block this, right?), just saw this when I was checking what improvements you made to learn from.

No intent to offend you.